### PR TITLE
COM-19900: Compress svg files

### PR DESCRIPTION
**Pull request by SylvainGuittard**
jira issue: https://jira.ez.no/browse/COM-19900

Tool used: https://compressor.io/compress
because http://www.scriptcompress.com/SVG-minifier.htm is down

### CODE REVIEW

**Comment by SylvainGuittard** [comment]
ping @clash82 @damianz5

File: web/assets/images/ez-logo.svg [hunk 1]
```diff
@@ -1,42 +1,13 @@
...
-					c4.3,4.5,9.5,8.1,15.4,10.8c6,2.6,12.7,4,19.8,4c4.7,0,9.4-0.6,13.8-1.8c4.4-1.2,8.4-2.9,12-5.1c3.6-2.1,6.8-4.7,9.6-7.6
-					c2.8-2.9,5-6.2,6.6-9.8l0.5-1.1l-15.7-4.3L258.1,131.7z M231.9,75.2c4,0,7.9,0.7,11.5,2.2c3.6,1.4,6.8,3.5,9.5,6
-					c2.7,2.5,5,5.7,6.8,9.3c1.6,3.3,2.7,7,3.1,10.9h-61.5c0.4-3.9,1.4-7.5,3-10.8c1.7-3.6,3.9-6.7,6.6-9.3c2.7-2.6,5.9-4.6,9.5-6.1
-					C223.9,75.9,227.8,75.2,231.9,75.2z"/>
-				<polygon points="403.2,28.8 289.1,28.8 289.1,45.8 367.7,45.8 275.9,159.7 391.2,159.7 391.2,142.7 311.5,142.7 				"/>
-			</g>
-		</g>
-	</g>
-</g>
+<!DOCTYPE svg  PUBLIC '-//W3C//DTD SVG 1.1//EN'  'http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd'>
```
Comment (damianz5) [inline web/assets/images/ez-logo.svg]: this line could be deleted

**Comment by damianz5** [comment]
Beside the comment +1

**Comment by clash82** [comment]
I found a new tool, can you try it?
https://jakearchibald.github.io/svgomg/

**Comment by SylvainGuittard** [comment]
Best tool!
PR updated

File: web/assets/images/ez-logo-white.svg [hunk 2]
```diff
@@ -1,27 +1 @@
...
-			c0.3-0.7,0.8-1.4,1.3-1.9c0.5-0.5,1.2-0.9,1.9-1.2C44.7,13,45.5,12.9,46.3,12.9z"/>
-		<polygon class="st0" points="80.8,3.6 57.8,3.6 57.8,7 73.6,7 55.2,29.9 78.4,29.9 78.4,26.5 62.3,26.5 		"/>
-	</g>
-</g>
-</svg>
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 82 31"><style>.st0{fill:#FFFFFF;} .st1{fill:#F05A22;}</style><path class="st0" d="M9.4 9h16.4V.9H1.2v24.6h8.2z"/><path class="st0" d="M11.4 11h18.9v18.9H11.4z"/><path class="st1" d="M13.7 13.3h14.4v14.4H13.7z"/><path class="st0" d="M51.6 24.3c-.2.5-.5.9-.8 1.2-.4.4-.8.7-1.2.9-.5.3-1 .5-1.5.6-.5.2-1.1.2-1.6.2-.8 0-1.6-.2-2.3-.5s-1.4-.7-1.9-1.3c-.6-.5-1-1.2-1.4-1.9-.3-.7-.5-1.5-.6-2.3h15.6V21c0-.1 0-.3.1-.5v-.7c0-1.3-.2-2.5-.7-3.7s-1.1-2.2-1.9-3.1-1.9-1.6-3.1-2.2c-1.2-.5-2.5-.8-4-.8-1.4 0-2.8.3-4 .8S40 12.1 39.2 13c-.9.9-1.6 2-2 3.2-.5 1.2-.7 2.5-.7 3.8s.2 2.6.7 3.9c.5 1.2 1.2 2.3 2 3.2.9.9 1.9 1.6 3.1 2.2 1.2.5 2.5.8 4 .8 1 0 1.9-.1 2.8-.4.9-.2 1.7-.6 2.4-1s1.4-.9 1.9-1.5c.6-.6 1-1.3 1.3-2l.1-.2-3.2-.9v.2zm-5.3-11.4c.8 0 1.6.1 2.3.4s1.4.7 1.9 1.2 1 1.1 1.4 1.9c.3.7.5 1.4.6 2.2H40.1c.1-.8.3-1.5.6-2.2s.8-1.4 1.3-1.9 1.2-.9 1.9-1.2c.8-.3 1.6-.4 2.4-.4zM80.8 3.6h-23V7h15.8L55.2 29.9h23.2v-3.4H62.3z"/></svg>
```
Comment (clash82) [inline web/assets/images/ez-logo-white.svg]: you can remove `id="Layer_1"`

**Comment by clash82** [comment]
Besides the above +1

**Pull request merged**